test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -191,8 +192,8 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 	// return blockID - top level block id which is looks like `block-b91ce479-fb2d-45b7-ad92-22ae7a58cf04`. Should be used for further interaction with added block.
 	async addBlock( name ) {
 		name = name.charAt( 0 ).toUpperCase() + name.slice( 1 ); // Capitalize block name
-		let blockClass = name;
-		let selectedBlockConfirmClass = 'is-selected';
+		let blockClass = kebabCase( name );
+		let hasChildBlocks = false;
```

Comment:
I've been trying to look for other existing blocks with inner blocks, and it may not always be the case that they get a "has-child-selected" when inserted e.g. "Layout Elements -> Media & Text" gets marked "is-selected" when inserted despite having an inner block. That said, no big deal - this looks good. 😃